app/src/test/java/src/Pieces/KnightMoveTest.java

```diff
@@ -1,38 +1,112 @@
-package src;
+package src.Pieces;
 
 import org.junit.Test;
-import src.Pieces.Piece;
-
 import java.util.List;
+import src.*;
 
 import static org.junit.Assert.assertEquals;
 
 public class KnightMoveTest {
+
+    private void chessboardScenario() {
+        Chessboard chessboard = new Chessboard();
+        int dimensionBoard = 8;
+        chessboard.board = new Piece[dimensionBoard][dimensionBoard];
+        GetPieceFactory getPieceFactory = new GetPieceFactory();
+        chessboard.board[2][4] = getPieceFactory.getPiece(TypePiece.HORSE, true, new Position(4,2));
+        chessboard.board[5][6] = getPieceFactory.getPiece(TypePiece.PAWN, false, new Position(6,5));
+        chessboard.board[6][2] = getPieceFactory.getPiece(TypePiece.ROOK, true, new Position(2,6));
+        chessboard.board[5][1] = getPieceFactory.getPiece(TypePiece.BISHOP, false, new Position(1,5));
```

Comment:
I suggest to use the merhod setPiece, like that
Chessboard.setPiece(getPieceFactory.getPiece(TypePiece.BISHOP, false, new Position(1,5));

Reply:
You are right, I will improve that.